features/step_definitions/user_authentication_steps.rb

```diff
@@ -3,3 +3,13 @@
     FactoryGirl.create(:user, username: user[:username], email: user[:email], password: 'password')
   end
 end
+
+Given(/^I am signed in$/) do
+  steps %Q{
+    Given I am on the "login page"
+    And I fill in "Email" with "calle@gmail.com"
+    And I fill in "Password" with "password"
+    And I click "Log in"
+    Then I should be on the "home page"
+  }
```

Comment:
This way of logging in a user is perfectly fine. There is, however, another, faster way using the Devise helpers. I showcased that in the Rails messaging project. 

Not a blocker though. You can keep this.